### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by adamdavenport** [aristocratos/bpytop#272] [Ideas]
I like the monitoring utility that you have created. I would like to see a feature that allows users to customize the order that the modules are displayed. For instance, swap networking with the cpu monitor.

**Comment by aristocratos** [comment]
@adamdavenport 
Because of how I wrote the first implementation of the boxes (with speed in mind), it's a lot of rewriting needed to make them swappable since the adaptable sizing is based on the boxes being in fixed positions.

I will get to it at some point, but it's unlikely to be soon.

**Reply by adamdavenport** [reply]
@aristocratos, thank you for replying to my fr. I understand the re-write issue. I've been there myself. If it never happens, no worries. Best of luck with this and your other projects. Cheers!
